restfulpy/tests/test_jwt_cli.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+import pytest
+from bddcli import Given, given, when, stdout, stderr, Application
+from itsdangerous import TimedJSONWebSignatureSerializer
+from itsdangerous.exc import SignatureExpired
+from nanohttp import settings
+
+from restfulpy import Application as RestfulPyApplication
```

Comment:
I think `restfulpy` is one word. `RestfulpyApplication` is better than `RestfulPyApplication`.